common/templates/common/sample_djangre.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base_templates/base.html' %}
```

Comment:
⚠️ This file (as well as the other things involved in serving) should have a `{# TODO: delete me; this is just a reference example #}` comment